src/Controller/BulkController.php

```diff
@@ -308,4 +308,22 @@ public function getErrors(): array
     {
         return $this->errors;
     }
+
+    /**
+     * Get object type, when it is abstract
+     *
+     * @param string $id The object ID
+     * @param string $type The object type
+     * @return string
+     */
+    protected function getType(string $id, string $type): string
+    {
+        $schema = $this->Schema->getSchema($type);
```

Comment:
We don't need the second argument `$type` since `$this->objectType` is always used

We could also use `$this->Schema->abstractTypes()` in `initialize` method to see if `$this->objectType` is abstract just once and use a class attribute like `$abstractType` 

```php
protected function getType(string $id): string
{
    if (!$this->abstractType) {
        return $this->objectType;
     }
     // retrieve type via API 
}